requirements.txt

```diff
@@ -1,22 +1,64 @@
+antlr4-python3-runtime==4.8
+beautifulsoup4==4.11.1
+bitarray==2.6.0
 certifi==2022.6.15
 cffi==1.15.1
 charset-normalizer==2.1.1
+click==8.1.3
+colorama==0.4.6
+cycler==0.10.0
+Cython==0.29.32
+distlib==0.3.6
+fairseq==0.12.2
+ffmpeg-python==0.2.0
+filelock==3.8.0
 flake8==5.0.4
+future==0.18.2
 grpcio==1.47.0
 grpcio-reflection==1.47.0
 grpcio-tools==1.47.0
+huggingface-hub==0.11.0
+hydra-core==1.0.7
 idna==3.3
+joblib==1.2.0
+kiwisolver==1.3.2
+lxml==4.9.1
+matplotlib==3.4.3
 mccabe==0.7.0
+more-itertools==9.0.0
+numpy==1.23.5
+omegaconf==2.0.6
+packaging==21.3
+Pillow==8.3.2
+platformdirs==2.5.2
+portalocker==2.6.0
 protobuf==3.20.1
 pycodestyle==2.9.1
 pycparser==2.21
 pyflakes==2.5.0
+pyparsing==2.4.7
+python-dateutil==2.8.2
 python-dotenv==0.21.0
 PyYAML==6.0
+regex==2022.9.13
 requests==2.28.1
+sacrebleu==2.3.1
+sacremoses==0.0.53
+scipy==1.9.3
 six==1.16.0
+soupsieve==2.3.2.post1
+SpeechRecognition==3.8.1
 srt==3.5.2
+tabulate==0.9.0
+tensorboardX==2.5.1
+tokenizers==0.13.2
+torch==1.13.0
+torchaudio==0.13.0
 tqdm==4.64.0
+transformers==4.24.0
+typing_extensions==4.4.0
 urllib3==1.26.12
+virtualenv==20.16.5
 vosk==0.3.44
 websockets==10.3
+whisper @ git+https://github.com/openai/whisper.git@eff383b27b783e280c089475852ba83f20f64998
```

Comment:
Using this ref requires git to be installed in the docker container

Reply:
I just realized that the requirements.txt is broken. 🧐 
I will look into the needed dependencies and update the Dockerfile accordingly.

Reply:
The issue should be resolved now. _Dockerfile_ + _requirements.txt_ is updated and should work.